examples/assets/templates/start_template.j2

```diff
@@ -0,0 +1,10 @@
+Hello, {{ user.first_name }}!
+This is a helper bot, I sit in chat and help admins!
+Bot commands:
+/start — Hello message
```

Comment:
Its better to show jinja power here like in render_from_string example. A cycle can be made for instance:
```jinja2
{% for command in commands %}
/{{command.name}} - {{command.description}}
{% endfor %}
```